src/SConscript

```diff
@@ -41,7 +41,7 @@ import os
 HEADERS = ['api.h', 'iv.h', 'types.h', 'swim.h']
 HEADERS_GURT = ['dlog.h', 'debug.h', 'common.h', 'hash.h', 'list.h',
                 'heap.h', 'errno.h', 'fault_inject.h', 'debug_setup.h',
-                'types.h']
+                'types.h', 'atomic.h']
```

Comment:
You need to port the configure check that sets HAVE_STDATOMIC as well, I think

Reply:
Agree. Thanks!